### ISSUE TITLE
Event-driven app actions

### ISSUE THREAD

**Issue by gitcips** [iadgov/WALKOFF#163]
Hello.

I was able to set up a new app, with event driven action. It works fine, so far so good.
I noticed that the POST request always returns a OK 200, but starts the workflow only if the workflow itself has been started from the playbook. Is this the expected behavior?
How can I start the workflow at the POST event? Trigger? 
Thanks,

**Comment by JustinTervala** [comment]
Yes, the an event-driven step is meant to pause a workflow and resume it when an event occurs, but if the workflow is not running, then it cannot resume anything. I assume by saying it is returning a 200, you are using a server endpoint to trigger the event. This is in the demo, but it is not required. If you want to spawn a thread or somehow else monitor a process until a trigger occurs, then you can do that as well. As for the server response, behind the scenes you are plugging your endpoint into a Flask blueprint, so you can customize the server response with whatever Flask will allow you to do. In particular, if you want an endpoint to response with some value, you can just return 
` return {'some data': 'you want returned'}, 201` And it'll return the data and a 201 HTTP status code. 

You can use a trigger to start a workflow from the REST API.
